automation-api/lib/ai_eval_spreadsheet/schemas.py

```diff
@@ -102,3 +99,37 @@ class GenAiModelsDf(pa.DataFrameModel):
     class Config:
         dtype = PydanticModel(GenAiModel)
         coerce = True
+
+
+class GenAiModelConfig(BaseModel):
+    include_in_next_evaluation: Optional[bool] = Field(
+        None, title="Include in next evaluation"
+    )
+    model_config_id: Optional[str] = Field(None, title="Model configuration ID")
+    model_id: Optional[str] = Field(None, title="Model ID")
+    model_parameters: Optional[str] = Field(None, title="Model Parameters")
+    repeat_times: Optional[int] = Field(None, title="Repeat Times")
+
+
+class GenAiModelConfigsDf(pa.DataFrameModel):
+    class Config:
+        dtype = PydanticModel(GenAiModelConfig)
+        coerce = True
+
+
+class EvalResult(BaseModel):
+    question_id: Optional[str] = Field(None, title="Question ID")
+    prompt_variation_id: Optional[str] = Field(None, title="Prompt variation ID")
+    model_configuration_id: Optional[str] = Field(None, title="Model Configuration ID")
+    last_evaluation_time: Optional[float] = Field(None, title="Last Evaluation Time")
```

Comment:
We can use a datetime type here (https://docs.pydantic.dev/latest/usage/types/#datetime-types) so that we use human and machine readable isoformatted datetime representations.

```suggestion
    last_evaluation_datetime: Optional[datetime] = Field(None, title="Last Evaluation")
```